src/tools/java/mekanism/tools/common/item/ItemMekanismPaxel.java

```diff
@@ -96,28 +96,33 @@ public InteractionResult useOn(UseOnContext context) {
         BlockPos blockpos = context.getClickedPos();
         Player player = context.getPlayer();
         BlockState blockstate = world.getBlockState(blockpos);
```

Comment:
This should be below attempting to use it as an axe as we unfortunately can't make use of it and super will calculate it anyway, but then at least if we can use it as an axe we only have to look up the block state once instead of twice.